user/zerotier.rst

```diff
@@ -0,0 +1,131 @@
+ZeroTier tunnel automation
+--------------------------
+
+In this guide we will explore how to set up the automatic management
+of `ZeroTier tunnels. <https://www.zerotier.com/>`_
+
+.. contents:: **Table of Contents**:
+   :backlinks: none
+   :depth: 3
+
+How to setup ZeroTier Tunnels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the procedure described below to setup
+ZeroTier tunnels on your devices.
+
+**Note:** This example uses **Shared systemwide (no organization)**
+option as the organization for VPN server and VPN client template.
+You can use any organization as long as VPN server, VPN client
+template and Device has same organization.
```

Comment:
Not to be done in the this PR, but perhaps we can move this boilerplate to a file and link that file  in all the docs!

Reply:
I'll create issue for that, thanks!